config/mail.php

```diff
@@ -60,6 +60,11 @@
         'name' => env('MAIL_FROM_NAME', 'Example'),
     ],
 
+    'to_admin' => [
+        'address' => env('MAIL_TO_ADMIN_ADDRESS', 'hello@example.com'),
```

Comment:
aside: defaults when a user hasn't bothered with the relevant .env entries, for some reason;
Should these resolve to something we 
a) know should exist (postmaster@localhost?)
b) know should never exist (noreply@example.com)
c) something that demo's what we expect (voucheradmin@example.com) in case anyone's checking the code for help.
I'm tempted by b)